pkg/server/helpers_selinux_test.go

```diff
@@ -65,7 +85,12 @@ func TestInitSelinuxOpts(t *testing.T) {
 		t.Logf("TestCase %q", desc)
 		processLabel, mountLabel, err := initSelinuxOpts(test.selinuxOpt)
 		assert.NoError(t, err)
-		assert.Equal(t, test.processLabel, processLabel)
-		assert.Contains(t, test.mountLabels, mountLabel)
+		if desc == "testUser" {
```

Comment:
What about here?

I don't think we want to change test behavior based on description. I'd rather add a filed in the struct to indicate whether we should use equal to do regexp match.

Reply:
Sorry, should change `testUser` to `Should be resolved correctly when selinuxOpt has been initialized completely`.

Reply:
Let's add field in the struct instead.

Reply:
>I'd rather add a filed in the struct to indicate whether we should use equal to do regexp match.
That's better. :)